src/python/WMCore/Cache/WMConfigCache.py

```diff
@@ -299,7 +304,7 @@ def loadAttachment(self, name, overwrite=True):
         attach = self.database.getAttachment(self.document["_id"], name)
 
         if not overwrite:
-            if name in self.attachments.keys():
+            if name in list(self.attachments):
```

Comment:
How about
```suggestion
            if name in self.attachments:
```
?

Reply:
You are definitely right, thank you very much!